pytest_instafail.py

```diff
@@ -46,7 +46,8 @@ def pytest_collectreport(self, report):
         # Show errors occurred during the collection instantly.
         TerminalReporter.pytest_collectreport(self, report)
         if report.failed:
-            self.rewrite("")  # erase the "collecting" message
+            if self.isatty:
```

Comment:
Well, 2.8 is not tested - the most recent version is 2.6.1.. ^^
Updated this PR and created a new one to add 3.0.2.